### DISCUSSION TITLE
Hao deng-uniled-controller

### DISCUSSION THREAD

**Discussion by bilandzijalive** [monty68/uniled#66] [General]
dear, do you know how to fix this error?
![IMG_2817](https://github.com/monty68/uniled/assets/132263687/4ec4b3ac-3677-4608-88dd-b7722c82f9bc)

**Comment by monty68** [comment]
If you enable debugging of Uniled and try again, then send me a copy of the log. It should help me to identify the issue. Note, ensure you sanitise the log before sending it as it will show your account details, and you need to hide them.
